### ISSUE TITLE
asciidoctor-diagram generated images not shown after release 0.5.0

### ISSUE THREAD

**Issue by crra** [joaompinto/asciidoctor-vscode#67]
Hello,

I'm using the ruby asciidoctor backend with the asciidoctor-diagram extension:

```
"AsciiDoc.use_asciidoctor_js": false,
"AsciiDoc.asciidoctor_command": "asciidoctor -r asciidoctor-diagram -o-",
```

After the 0.5.0 release, the images are generated but not longer shown in the preview.

**Comment by joaompinto** [comment]
Fixed in 0.7.1, thanks for reporting.

**Comment by thetric** [comment]
Just stumbled into this issue, too. Thank you for the very fast bug fix :+1:
